integration/src/main/java/scripts/utils/RetryUtils.java

```diff
@@ -0,0 +1,99 @@
+package scripts.utils;
```

Comment:
Took this util class from the service project. Do we have a better pattern of retry + assert?

Reply:
for later - this might make sense in TCL or wsm- common lib(?)